explore/src/main/scala/explore/targets/TargetSearchResult.scala

```diff
@@ -30,7 +30,7 @@ object TargetSearchResult {
         none,
         Target.Sidereal.sourceProfile
           .andThen(SourceProfile.unnormalizedSED)
-          .modify(_.orElse(Constants.DefaultSED.some))(r.target) // Ensure SED is always present.
+          .modify(identity)(r.target)
```

Comment:
Can all this be replaced with just `r.target`?

Reply:
Right